doc/FAQ.md

```diff
@@ -16,4 +16,20 @@ Further more you will also have choices in the HttpClient that is utilized.
 
 By default kubenetes-client has a runtime dependency on OkHttp (kubernetes-httpclient-okhttp).  If you need to directly manipulate OkHttp, you add a compile dependency to kubernetes-httpclient-okhttp.  
 
-If you wish to use another HttpClient implementation typically you will exclude kubernetes-httpclient-okhttp and include the other runtime dependency instead.
\ No newline at end of file
+If you wish to use another HttpClient implementation typically you will exclude kubernetes-httpclient-okhttp and include the other runtime dependency instead.
+
+### What threading concerns are there?
+
+There has been a lot of changes under the covers with thread utilization in the fabric8 client over the 5.x and 6.x releases.  So the exact details of what threads are created / used where will depend on the particular release version.
+
+At the core the thread utilization will depend upon the http client implementation.  Per client OkHttp maintains a pool of threads for task execution.  It will dedicate 2 threads out of that pool per WebSocket connection.  If you have a lot of WebSocket usage (Informer or Watches) with OkHttp, you can expect to see a large number of threads in use - which can potentially exhaust the OkHttp defaults.
+
+With the JDK http client it will only maintain a selector thread and a small worker pool which will be based upon your available processors per client.  It does not matter how many Informers or Watches you run, the same worker pool is shared.
+
+It is recommended with either http client that logic you supply via Watchers, ExecListeners, ResourceEventHandlers, Predicates, etc. do not execute long running tasks. 
```

Comment:
I'd try to highlight this as a callout note box:

```suggestion
> **Note:** It is recommended with either HTTP client implementation that logic you supply via Watchers, ExecListeners, ResourceEventHandlers, Predicates, etc. does not execute long running tasks. 
```

Maybe also suggest the appropriate approach too ---> Schedule those long-running tasks in a separate executor

Reply:
> Maybe also suggest the appropriate approach too ---> Schedule those long-running tasks in a separate executor

A separate task queue is mentioned below.  We can move or elaborate on that if you want.

Reply:
No, it's fine, I saw the additional explanation later.
The callout is good for diagonal readers, which might stop and pay closer attention to the surrounding paragraphs.